src/BenchmarksApps/TechEmpower/Mvc/mvc.benchmarks.yml

```diff
@@ -30,17 +30,6 @@ jobs:
     noClean: true
 
 scenarios:
-  fortunes:
```

Comment:
Why removed?

Reply:
Was duplicated, see further down in file.